### PR TITLE
Decouple from Rocket.Chat repo

### PR DESCRIPTION
**Pull request by sampaiodiego**
- [x] Replace `getEngineVersions` by a `getMetadata` function defined on the original repo's `package.json` file property `houston.metadata`
- [x] Replace [engine versions markdown generator](https://github.com/RocketChat/Rocket.Chat.Houston/blob/99ead3c9c95b3e8abdddfc3dcbf59d2caba8cc72/src/md.js#L279) by a "custom markdown" defined on the original repo's `package.json` file property `houston.markdown`
- [x] Support `minTag` config

### CODE REVIEW

File: src/set-version.js [hunk 1]
```diff
@@ -497,7 +513,7 @@ class Houston {
 			name: 'pushTag'
 		}]);
 
-		const body = await md({tag: this.version, write: false, title: false});
+		const body = await md({tag: this.version, write: false, title: false, owner: this.owner, repo: this.repo});
```
Comment (rodrigok) [inline src/set-version.js:516]: Should pass `customMarkdown: this.customMarkdown`?

File: src/set-version.js [hunk 2]
```diff
@@ -537,7 +553,7 @@ class Houston {
 			name: 'create'
 		}]);
 
-		const body = await md({tag: this.version, write: false, title: false});
+		const body = await md({tag: this.version, write: false, title: false, owner: this.owner, repo: this.repo});
```
Comment (rodrigok) [inline src/set-version.js:556]: Should pass `customMarkdown: this.customMarkdown`?

File: src/set-version.js [hunk 3]
```diff
@@ -560,7 +576,7 @@ class Houston {
...
```
Comment (rodrigok) [inline src/set-version.js:579]: Should pass `customMarkdown: this.customMarkdown`?

File: src/logs.js [hunk 4]
```diff
@@ -246,63 +245,28 @@ async function getTags() {
 				before: index ? tags[--index] : null
 			};
 		})
-		.filter(item => item.tag === 'HEAD' || semver.gte(item.tag, minTag))
+		.filter(item => item.tag === 'HEAD' || (minTag && semver.gte(item.tag, minTag)))
```
Comment (rodrigok) [inline src/logs.js:248]: Does this means that without `minTag` it will filter all tags out?

Comment (sampaiodiego) [reply src/logs.js:248]: oopsie

File: src/set-version.js (hunk 1 above)
Comment (sampaiodiego) [reply src/set-version.js:516]: yes

**Comment by sampaiodiego** [comment]
closing in favor of https://github.com/RocketChat/Rocket.Chat.Houston/pull/29

**Pull request closed without merging**